tests/foreman/ui/test_hostcollection.py

```diff
@@ -26,14 +26,14 @@
 
 @pytest.fixture(scope='module')
 def module_manifest():
-    with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
+    with Manifester(manifest_category=settings.manifest.golden_ticket) as manifest:
         yield manifest
 
 
 @pytest.fixture(scope='module')
 def module_org_with_parameter(module_target_sat, module_manifest):
     # adding remote_execution_connect_by_ip=Yes at org level
-    org = module_target_sat.api.Organization(simple_content_access=False).create()
+    org = module_target_sat.api.Organization(simple_content_access=True).create()
```

Comment:
```suggestion
    org = module_target_sat.api.Organization().create()
```
SCA should be default for new orgs.

Reply:
We should add BlockedBy to docstring since there is bug reported
```
:BlockedBy: SAT-25817
```